posts/api_enabled_retail.md

```diff
@@ -0,0 +1,67 @@
+# Modern APIs are Pushing Innovation in Retail
+
+An increasing number of retailers are API enabling core parts of their business, delivering a more cohesive customer experience across a variety of touchpoints, making it easier to streamline internal operations, and opening opportunities to interface with external channels.
+
+## What is an API?
+
+First off, it's worth recapping on what we mean by API, or Application Programming Interface, to use its full name.
+
+An API allows systems to easily talk to one another. APIs can allow an e-commerce store to send an order to the warehouse for fulfilment, or an EPOS system understand what stock is available in the store.
+
+More modern applications are often API ready out of the box, but legacy applications can sometimes cause sticking points.
+
+## Legacy Applications and Proprietary Protocols
+
+Historically, applications connected via proprietary protocols. Different vendors had their own standards, and connecting different systems was often a laborious exercise that involved licensing more proprietary software.
+
+Worse is that some software systems are "walled gardens", where they by design, or through lack of investment, won't interoparate with other systems. Isolating their data and functionality from the rest of the organisation.
```

Comment:
I would shift some commas here: ...are "walled gardens" where they, by design or through lack of investment, won't...

Also: interoperate typo